android/src/main/java/com/mobilejazz/harmony/kotlin/android/ui/base/BaseActivity.kt

```diff
@@ -12,10 +12,19 @@ import javax.inject.Inject
 
 abstract class BaseActivity : AppCompatActivity(), HasSupportFragmentInjector {
 
-  @Inject lateinit var fragmentInjector: DispatchingAndroidInjector<Fragment>
+  @Inject
+  lateinit var fragmentInjector: DispatchingAndroidInjector<Fragment>
+
+  open fun injectDependencies() {
+    try {
+      AndroidInjection.inject(this)
+    } catch (t: Throwable) {
+    }
```

Comment:
I would suggest to add a small log saying that Dagger is not configure just in case that you are using dagger and you don't know what's happening.